### PR TITLE
Hi John kani this side, we talked on discord about looping issue in my plugin, Please look into it!

### PR DESCRIPTION
**Pull request by sonikanishk**

### CODE REVIEW

**Comment by JohnMcLear** [comment]
See linting errors, please resolve those before I take a look :)  They might fix the bug ;)

File: static/js/index.js [hunk 1]
```diff
@@ -1292,9 +1295,77 @@ const hooks = {
...
     }
+    if (!context.callstack.docTextChanged) {
+      return cb();
+    }
+    const ace = context.editorInfo.editor;
+    const padlines = ace.exportText().split('\n');  
+    let i = 0;
+    const result = [];
+    if (padlines){
+      for (i = 0; i < padlines.length; i += 1){
```
Comment (JohnMcLear) [inline static/js/index.js:1306]: ``+=`` is for concatenation not incrementing.  Do ``i++``

????

Comment (sonikanishk) [reply static/js/index.js]: yes Thank you!
correcting it

**Comment by JohnMcLear** [comment]
Hi @sonikanishk - please can you resolve your merge conflicts 🥇 <3

**Pull request closed without merging**